pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -80,6 +80,8 @@
 , withOpus ? withHeadlessDeps # Opus de/encoder
 , withPlacebo ? withFullDeps && !stdenv.isDarwin # libplacebo video processing library
 , withPulse ? withSmallDeps && stdenv.isLinux # Pulseaudio input support
+, withQrencode ? withFullDeps && lib.versionAtLeast version "7" # QR encode generation
+, withQuirc ? withFullDeps && lib.versionAtLeast version "7" # QR decoding
```

Comment:
These add ~20M to the closure of small which is quite a bit for such a fringe feature. Let's keep this in full for now.